pkgs/development/tools/profiling/systemtap/default.nix

```diff
@@ -14,7 +14,12 @@ let
   stapBuild = stdenv.mkDerivation {
     name = "systemtap-${version}";
     src = fetchgit { inherit url rev sha256; };
-    buildInputs = [ elfutils pkgconfig gettext ];
+    buildInputs = [ elfutils pkgconfig gettext python2 pythonPackages.setuptools ];
+    postPatch = ''
+      substituteInPlace buildrun.cxx --replace \
+        'o << "EXTRA_CFLAGS += -I\"" << s.runtime_path << "\"" << endl;' \
+        'o << "EXTRA_CFLAGS += -I" << s.runtime_path  << endl;'
+    '';
```

Comment:
I was just about to submit a PR on this aswell, but your workaround here is more minimal (I fixed it by recreating a full kbuild dir).  Could you add a comment though? Something like `FIXME: Workaround for bug in kbuild, where quoted -I"/foo" flags would get mangled in out-of-tree kbuild dirs`